package-release.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 if [ -z "$1" ]; then
-  echo "Usage: package-release.sh destdir"
+  echo "Usage: package-release.sh destdir [--fakedlls]"
   exit 1
 fi
 
```

Comment:
While we are fixing the scripts, the line just below needs a change to prevent word splitting:
```bash
NVOPTIX_SRC_DIR=$(dirname "$(readlink -f "$0")")
```